rebac-admin-backend/v1/interfaces/capabilities.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2024 Canonical Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package interfaces
+
+import (
+	"context"
+
+	"github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
+)
+
+// CapabilitiesService defines an abstract backend to handle capabilities related operations.
+type CapabilitiesService interface {
+	ListCapabilities(ctx context.Context) (*resources.Capabilities, error)
```

Comment:
We don't need an interface for the `/capabilities` endpoint. This one endpoint should be handled by the library (based on non-`nil` service backends provided by the product services).

Reply:
Yeah, I thought of having fine-grained capabilities support before, but that seemed a bit irrelevant. However, your example is not that far from reality. So, let's rephrase my comment. Let's provide some mechanism for the product service to inform us on the capabilities, and then we translate it to the OpenAPI spec response in the handler. How about that?

Reply:
I think the simplest way to do that is using the same interface approach we used until now.
That is in all effect the mechanism you refer to, since it lets product teams provide that info. Any other solution would probably be "over-designing" at this point.

Reply:
I resolve this comment, as we decided to merge it as is and polish it later, with more context/ideas from other.